yolov5/yolov5.cpp

```diff
@@ -62,26 +62,22 @@ ICudaEngine* build_engine(unsigned int maxBatchSize, IBuilder* builder, IBuilder
     auto bottleneck_csp9 = C3(network, weightMap, *spp8->getOutput(0), get_width(1024, gw), get_width(1024, gw), get_depth(3, gd), false, 1, 0.5, "model.9");
     auto conv10 = convBlock(network, weightMap, *bottleneck_csp9->getOutput(0), get_width(512, gw), 1, 1, 1, "model.10");
 
-    float* deval = reinterpret_cast<float*>(malloc(sizeof(float) * get_width(512, gw) * 2 * 2));
-    for (int i = 0; i < get_width(512, gw) * 2 * 2; i++) {
-        deval[i] = 1.0;
-    }
-    Weights deconvwts11{ DataType::kFLOAT, deval, get_width(512, gw) * 2 * 2 };
-    IDeconvolutionLayer* deconv11 = network->addDeconvolutionNd(*conv10->getOutput(0), get_width(512, gw), DimsHW{ 2, 2 }, deconvwts11, emptywts);
-    deconv11->setStrideNd(DimsHW{ 2, 2 });
-    deconv11->setNbGroups(get_width(512, gw));
-    weightMap["deconv11"] = deconvwts11;
+    auto upsample11 = network->addResize(*conv10);
+    assert(upsample11);
+    upsample11->setResizeMode(ResizeMode::kNEAREST);
+    upsample11->setOutputDimensions(bottleneck_csp6->getDimensions());

```

Comment:
```
yolov5.cpp:65:49: error: no matching function for call to ‘nvinfer1::INetworkDefinition::addResize(nvinfer1::ILayer&)’
     auto upsample11 = network->addResize(*conv10);
```

Compile error.

Pls verify it can work before PR.
                                                 ^